### ISSUE TITLE
Animation flutter is not working properly

### ISSUE THREAD

**Issue by phuongkna79** [xvrh/lottie-flutter#275]
Dear @xvrh,

This animation is not working properly on flutter same issues: #243, link: https://lottiefiles.com/73194-happy-birthday (https://assets3.lottiefiles.com/private_files/lf30_kkceplqe.json).

Thanks!

**Comment by xvrh** [comment]
*I think* this is because the animation uses **expressions** which are not supported.

**Comment by zakiso** [comment]
Could anyone can be support **expressions** .  Same issue here

**Comment by srinivas-nahak** [comment]
I'm facing the same issue and I'm using the expression for "ping-pong" effect in After effects. Without the expression the animation is not looking smooth but by adding it, it's not playing properly in flutter.

**Comment by xvrh** [comment]
@srinivas-nahak in After effects you can convert your expression to keyframes.

**Comment by srinivas-nahak** [comment]
Yes after experimenting a lot, I did that exactly and now I got the desired effect.

@xvrh thanks for your suggestion :)
